config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -649,6 +649,85 @@ presubmits:
         - name: ndots
           value: "1"
 
+  - name: pull-cert-manager-make-e2e-v1-24
+    context: pull-cert-manager-make-e2e-v1-24
+    always_run: false
+    optional: true
+    max_concurrency: 4
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: jetstack-cert-manager-presubmits-blocking
+      description: Runs 'make e2e-ci K8S_VERSION=1.24'
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-cloudflare-credentials: "true"
+      preset-retry-flakey-tests: "true"
+      preset-enable-all-feature-gates: "true"
+      preset-ginkgo-skip-default: "true"
+    spec:
+      containers:
+        # TODO: change to a custom image that embeds the system tools we
```

Comment:
This should really be done soon, I _think_ we may be able to use [the bazel tools image](https://github.com/jetstack/testing/tree/master/images/bazel-tools) as that already has `jq` and `make`, but not sure whether we need any extra functionality to set up docker/ensure logs are exported etc